From e4443a43c3e92a5f6dc8a3a9edba443be314a819 Mon Sep 17 00:00:00 2001
From: Hiroshi Hatake <hatake@clear-code.com>
Date: Fri, 16 Jun 2017 16:33:13 +0900
Subject: [PATCH] Call fEGLImageTargetTexture2D eariler

fEGLImageTargetTexture2D is called in GLLibraryEGL::ReadbackEGLImage.
This method will be called after EGLImage created, but this patch calls
it eariler.
---
 gfx/gl/SharedSurfaceEGL.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gfx/gl/SharedSurfaceEGL.cpp b/gfx/gl/SharedSurfaceEGL.cpp
index be79644336ca..d0c2d28b99af 100644
--- a/gfx/gl/SharedSurfaceEGL.cpp
+++ b/gfx/gl/SharedSurfaceEGL.cpp
@@ -47,6 +47,7 @@ SharedSurface_EGLImage::Create(GLContext* prodGL,
         prodGL->fDeleteTextures(1, &prodTex);
         return Move(ret);
     }
+    prodGL->fEGLImageTargetTexture2D(LOCAL_EGL_GL_TEXTURE_2D, image);
 
     ret.reset( new SharedSurface_EGLImage(prodGL, egl, size, hasAlpha,
                                           formats, prodTex, image) );
-- 
2.11.0

